qiskit_nature/second_q/mappers/parity_mapper.py

```diff
@@ -14,27 +14,51 @@
 
 from __future__ import annotations
 
+import logging
+
 from functools import lru_cache
 from typing import Union, List
 
 import numpy as np
 
 from qiskit.opflow import PauliSumOp
-from qiskit.quantum_info.operators import Pauli
+from qiskit.quantum_info.analysis.z2_symmetries import Z2Symmetries
+from qiskit.quantum_info.operators import Pauli, PauliList, SparsePauliOp
 
 from qiskit_nature.second_q.operators import FermionicOp
 from .fermionic_mapper import FermionicMapper
 
+logger = logging.getLogger(__name__)
+
 
 class ParityMapper(FermionicMapper):  # pylint: disable=missing-class-docstring
-    def __init__(self):
+    def __init__(self, num_particles: tuple[int, int] | None = None):
         """The Parity fermion-to-qubit mapping.
 
         When using this mapper ``two_qubit_reduction`` can optionally be used for the qubit
         operator that is created, see converter class
         :class:`~qiskit_nature.second_q.mappers.QubitConverter`.
         """
         super().__init__(allows_two_qubit_reduction=True)
```

Comment:
Note for a future reviewer: we decided that the super init argument `allows_two_qubit_reduction` will be deprecated in a separate PR